content/en/apps/guides/database/_index.md

```diff
@@ -4,4 +4,50 @@ linkTitle: "Database"
 weight: 100
 description: >
   Managing databases used by CHT applications
----
\ No newline at end of file
+---
+
+## CouchDB
+
+The CHT has a range of CouchDB databases for storing different types of data. By default, databases all start with the prefix "medic".
+
+### medic
+
+The main database, used to store all patient and report data. Data access is protected by API to provide protection on a per document basis.
```

Comment:
Do all the items mentioned in the "_See Also_: Database schema conventions" reside in this database? Not saying we need to call them all out here, though we could say something to illustrate that there are other things in there besides just patient and report data.

Also, to be consistent, perhaps we should replace "patient" with "contact".

Reply:
> Do all the items mentioned in the "See Also: Database schema conventions" reside in this database?

Mostly. There's a brief mention of the _users db too. I was trying to draw a distinction between data (stored in medic) and metadata (stored in the user meta dbs).

> Also, to be consistent, perhaps we should replace "patient" with "contact".

Done.